example-workflows/cloud-run/cloud-run.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build and Deploy to Cloud Run
+
+on:
+  push:
+    branches:
+    - master
+
+# Environment variables available to all jobs and steps in this workflow
+env:
+  PROJECT_ID: ${{ secrets.RUN_PROJECT }}
+  RUN_REGION: us-central1
+  SERVICE_NAME: helloworld-nodejs
+
+jobs:
+  setup-build-deploy:
+    name: Setup, Build, and Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      # Setup gcloud CLI
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '275.0.0'
+          service_account_email: ${{ secrets.SA_EMAIL }}
+          service_account_key: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS}}
+
+      # Configure gcloud CLI
+      - name: gcloud Set up
+        run: |
+          gcloud config set project $PROJECT_ID
+          gcloud config set run/region $RUN_REGION
+
+      # Build and push image to Google Container Registry
+      - name: Build
+        run: |
+          gcloud builds submit -t gcr.io/$PROJECT_ID/$SERVICE_NAME:$GITHUB_SHA
+
+      # Deploy image to Cloud Run
+      - name: Deploy
+        run: |
+          gcloud run deploy $SERVICE_NAME \
+            --image gcr.io/$PROJECT_ID/$SERVICE_NAME:$GITHUB_SHA \
+            --platform managed \
+            --allow-unauthenticated
```

Comment:
Generally recommended to not include `--allow-unauthenticated` in your CD script.

Some considerations which may not apply to everyone's use case or testing constraints (including "easy of learning from an example")

* Requires broader privileges than regular Cloud Run deployment
* Removes the go/no-go decision from humans
* Locking down access is one step you might want to take in an emergency situation (e.g., data corruption), but this configuration will re-expose the service automatically.

If you do remove it, Cloud Run will persist this setting from revision to revision if it's used just a single time.

Reply:
Perhaps we should strive for consistency with the official getting started docs for this project on this matter: https://cloud.google.com/run/docs/deploying

Reply:
Deploying page does not state an opinion about the use of --allow-unauthenticated, and https://cloud.google.com/run/docs/continuous-deployment-with-cloud-build deliberately leaves it out for reasons of point 2.

Reply:
If you click on the "Command line" tab you'll see it's directly mentioned. @averikitsch would it be possible to follow suit and include that snippet in the README for this?

Reply:
I've removed the flag and added a note about how it works, how to find more information, and  how to change it.